api-specificatie/openapi.yaml

```diff
@@ -86,7 +83,77 @@ paths:
+        '406':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/406'
+        '410':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/410'
+        '412':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/412'
+        '429':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/429'
+        '500':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/500'
+        '503':
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/503'
+        default:
+          $ref: 'https://raw.githubusercontent.com/lvbag/BAG-API/master/Technische%20specificatie/Yaml%27s/20191209-1-lvbag-individuele-bevragingen-20180601/generiek/responses.yaml#/default'
+
+  /adressen:
+    get:
+      security:
+        - apiKeyBAG: []
+      operationId: raadpleegAdresMetKeuzeItemId
+      tags:
+        - Adres
+      summary: Raadpleeg adres met de identificatie van het geselecteerde item 
+      description: "Raadpleeg het adres met de identificatie van het geselecteerde item verkregen via de zoeken-operatie
+        Geleverd worden de huidige gegevens van een adres evt. met embedded woonplaats, openbare ruimte, nummeraanduiding. 
+        Huidige gegevens zijn actuele gegevens die geen eind status hebben, zoals 'Naamgeving ingetrokken'. 
+        Als de nummeraanduiding is ingetrokken, dan levert een bevraging geen adres op. 
+        Relaties met adresseerbaarobject (ligplaats, standplaats of verblijfsobject) en als het adresseerbaare object een verblijfsobject betreft het/de gerelateerde pand(en). 
+        Parameter keuzeItemIdentificatie bevat identificatie van een keuze item verkregen uit een zoek-operatie.
+        Parameter adresString bevat een weergavenaam zoals verkregen uit een zoek-operatie.
+        Parameter adresseerbaarobjectIdentificatie bevat de 16 cijferige identificatie van een ligplaats, standplaats of verblijfsobject."
+      parameters:
+        - $ref: '#/components/parameters/keuzeItemIdentificatie'
+        - $ref: '#/components/parameters/adresString'
+        - $ref: '#/components/parameters/adresseerbaarObjectIdentificatie-query'
```

Comment:
Waarom staat hier query? Het is een query parameter en is gedefinieerd als components-parameter. Graag -query weghalen.

Reply:
Er wordt bij één van de andere endpoints gezocht met een path parameter adresseerbaarObjectIdentificatie. Aangezien een parameter of een path of een query parameter moet zijn (en niet beide kan zijn), moeten er 2 parameters worden gedefinieerd, Johan had ervoor gekozen en post-fix '-query' te gebruiken voor query parameters en geen post-fix voor de path parameter voor hetzelfde gegeven. Ik heb dat overgenomen voor de andere gegevens waar zowel een path al query parameter nodig is. 
@CathyDingemanse als je een betere naamgeving weet voor path en query parameters voor een zelfde gegeven, dan hoor ik dat graag.
V.w.b. nummeraanduidingIdentificaie was toch afgesproken dat dit een unieke identificatie voor een adres is? Als dat zo is, dan is het een path parameter en geen query parameter. Juist om die reden is een aparte endpoint gemaakt waar met een nummeraanduidingIdentificatie een Adres opgevraagd kan worden. Die endpoint was al toegevoegd zoals bij View changes te zien is. De reden hiervoor was dus niet dat er mogelijk een nummeraanduidingIdentificatie terug zou komen in de zoekresultaten.